src/main/java/org/kainos/ea/api/RoleService.java

```diff
@@ -3,20 +3,14 @@
 import org.kainos.ea.cli.RoleRequest;
 import org.kainos.ea.cli.Role;
 
-import org.kainos.ea.client.FailedToCreateJobException;
-import org.kainos.ea.client.FailedToGetRolesException;
-import org.kainos.ea.client.InvalidJobException;
-import org.kainos.ea.client.ValidationException;
+import org.kainos.ea.client.*;
```

Comment:
this will fail checkstyle. Add staging-branch to maven.yml to get github action check to run